plugin.audio.rne/resources/tests/test_api.py

```diff
@@ -17,22 +17,22 @@ class ITTests(unittest.TestCase):
 
     def test_1_create_index(self):
         menu_list = api.get_create_index()
-        self.assertTrue(menu_list is not None and len(menu_list) == 4)
```

Comment:
Can you remove the test files next time? They are of no use for the add-on users.